packages/backend/src/models/ContentModel/ContentConfigurations/SqlChartContentConfiguration.ts

```diff
@@ -0,0 +1,188 @@
+import {
+    ChartContent,
+    ChartKind,
+    ChartSourceType,
+    ContentType,
+} from '@lightdash/common';
+import { Knex } from 'knex';
+import { DashboardsTableName } from '../../../database/entities/dashboards';
+import { OrganizationTableName } from '../../../database/entities/organizations';
+import { ProjectTableName } from '../../../database/entities/projects';
+import { SavedSqlTableName } from '../../../database/entities/savedSql';
+import { SpaceTableName } from '../../../database/entities/spaces';
+import { UserTableName } from '../../../database/entities/users';
+import {
+    ContentConfiguration,
+    ContentFilters,
+    SummaryContentRow,
+} from '../ContentModelTypes';
+
+type SelectSavedSql = SummaryContentRow<{
+    source: ChartSourceType.SQL;
+    chart_kind: ChartKind;
+    dashboard_uuid: string | null;
+    dashboard_name: string | null;
+}>;
+
+export const sqlChartContentConfiguration: ContentConfiguration<SelectSavedSql> =
```

Comment:
I thought this was going to be a class that extends an interface, but I guess this also works, and you don't need to initialize anything.

Reply:
Had the same thoughts and decided to skip the class boilerplate for now.